frontend/scss/globals/_g-header.scss

```diff
@@ -262,13 +276,18 @@
     padding-right: 4.6875vw;
     position: absolute;
     top: 24px;
-    visibility: collapse;
     @include breakpoint('medium-') {
       left: 67vw;
     }
   }
 }
 
+.g-header__nav-primary [role="menu"]:has(>li>a:hover, >li>a:focus, >li>a[aria-expanded="true"]) {
+  &>li>a:not(:hover, :focus, [aria-expanded="true"]) {
+    color: $color__black--30;
+  }
+}
```

Comment:
I also thought I'd have to do the dimming of non-hovered/-expanded menu items in JS, but was able to work it out with CSS, as well!